pkg/kubecost/allocation_test.go

```diff
@@ -2303,12 +2306,21 @@ func TestAllocationSetRange_AccumulateBy(t *testing.T) {
 
 			testId: "AccumulateBy Test 10",
 		},
+		{
+			asr:        NewAllocationSetRange(ago4dAS, ago3dAS, ago2dAS, yesterdayAS, todayAS),
+			resolution: time.Hour * 72,
+
+			expectedCost: 30.0,
+			expectedSets: 2,
+
+			testId: "AccumulateBy Test 11",
+		},
 	}
 
 	for _, c := range cases {
 		result, err = c.asr.AccumulateBy(c.resolution)
 		sumCost := 0.0
-
+		fmt.Println(c.testId)
```

Comment:
If you want to keep this, I'd recommend using `t.Logf` instead of `fmt.Println`. `t.Logf` messages won't show unless:
- the test fails OR
- you use `go test -v` instead of just regular `go test`

So, when debugging tests, you can use `go test -v` to get a more detailed picture of what's happening.

Reply:
Ah, I realize you removed this line. No need to add it back; this is just some nice info about `go test` 😄

Reply:
that's still useful, thanks!